stable_baselines/common/base_class.py

```diff
@@ -312,6 +357,51 @@ def action_probability(self, observation, state=None, mask=None, actions=None):
         """
         pass
 
+    def load_parameters(self, load_path_or_dict):
```

Comment:
as discuss in the issue, I would add an argument (`exact_match=True`) that when not deactivated will raise an error if the dict does not match with the expected one (e.g. missing keys).
This would prevent silent weird behavior.